app/lib/training_route_manager.rb

```diff
@@ -33,6 +33,12 @@ def itt_route?
     ITT_TRAINING_ROUTES.keys.any? { |training_route_enums_key| training_route == TRAINING_ROUTE_ENUMS[training_route_enums_key.to_sym] }
   end
 
+  def requires_study_mode?
+    return false if early_years_route?
+
+    %w[assessment_only pg_teaching_apprenticeship opt_in_undergrad].exclude?(training_route)
```

Comment:
Can we reference the enum keys directly to be consistent? eg `TRAINING_ROUTE_ENUMS[:assessment_only]`. It'll also make it easier to update the names from one place in the future